mpc/coordinator/powersoftau_process_command.py

```diff
@@ -45,4 +45,6 @@ def _exec(self, args: List[str]) -> bool:
 
 def _default_tool() -> str:
     from os.path import join, dirname
-    return join(dirname(__file__), "..", "..", "build", "src", "pot-process")
+    return join(
+        dirname(__file__), "..", "..",
+        "build", "mpc_tools", "pot-process")
```

Comment:
Similar to the above, was this intentional? We usually have all params on one line, or one-per-line.

Reply:
Fixed in https://github.com/clearmatics/zeth/pull/266/commits/98bedfa67af3f884f53373ec5a403ffdf737bccc